CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Unreleased
+
+### Dependencies
+
+- Bump Flutter SDK (metrics) from v3.3.3 to v3.3.4 ([#1063](https://github.com/getsentry/sentry-dart/pull/1063))
+  - [changelog](https://github.com/flutter/flutter/blob/main master/CHANGELOG.md#334)
+  - [diff](https://github.com/flutter/flutter/compare/3.3.3...3.3.4)
```

Comment:
@vaind since this is for testing only, is there a way to skip the changelog?

Reply:
Sure, I've had that in mind for a while just didn't need it yet. Makes sense here, let me prep Updater for that before merging this.